java/code/src/com/redhat/rhn/frontend/strings/database/StringResource_en_US.xml

```diff
@@ -1,6 +1,6 @@
-<?xml version="1.0" encoding="utf-8"?>
-<xliff xmlns="urn:oasis:names:tc:xliff:document:1.1" xmlns:xyz="urn:appInfo:Items" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="urn:oasis:names:tc:xliff:document:1.1 http://www.oasis-open.org/committees/xliff/documents/xliff-core-1.1.xsd" version="1.1">
-  <file source-language="en" datatype="plaintext" original="">
+<?xml version='1.0' encoding='utf-8'?>
```

Comment:
I'm not sure if this was an intentional change? The xml tag is the only one using single quotes at the moment, both here and in other files.

Reply:
We use a tool to generate this. So that change comes from the python library.